src/App.vue

```diff
@@ -12,7 +12,16 @@ import TheHeader from "@/components/TheHeader.vue";
 </template>
 
 <style scoped lang="scss">
+#app {
```

Comment:
I don't see any elements with id `app`. Did you mean to apply this piece of styling to class `app` instead?

Reply:
I am actually not sure how did I end up with `#app` and `.app`. I will tidy this up :)